app/controllers/admin/categories_controller.rb

```diff
@@ -43,12 +48,12 @@ def new_or_edit
   end
 
   def save_category
-    if @category.save!
+    @category.save
+    if @category.errors.empty?
```

Comment:
You can also utilize the `save` method without the bang in this conditional which should give you the same result